<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoScan</class>
 <widget class="QDialog" name="AutoScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>オートスキャン</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/assets/images/radar.png</normaloff>:/images/assets/images/radar.png</iconset>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>350</width>
     <height>100</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>174</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>IPアドレス</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ポート</string>
    </property>
   </column>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>125</y>
     <width>350</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>160</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>オートスキャン</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>確認</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>160</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>キャンセル</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>AutoScan</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
